src/sasctl/_services/model_repository.py

```diff
@@ -667,15 +670,37 @@ def copy_python_resources(cls, model):
         """
         if cls.is_uuid(model):
             id_ = model
+            model = cls.get_model(model)
         elif isinstance(model, dict) and 'id' in model:
             id_ = model['id']
+            model = cls.get_model(model)
         else:
             model = cls.get_model(model)
             id_ = model['id']
 
-        return cls.put(
-            '/models/%s/scoreResources' % id_, headers={'Accept': 'application/json'}
-        )
+        # Check if symbolic link for resource directories exists
+        try:
```

Comment:
Service classes should generally behave like the underlying service calls.  This overwrite functionality should probably be moved somewhere else.  pzmm? utils?

Reply:
I created two new functions (model_exists and project_exists) within ImportModel.py that carries out the additional functionality outside of the service calls. For now, they exist solely within pzmm, but they may find a better home in utils or elsewhere.